File: 0268. Missing Number/solution.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        summation = (len(nums) * (len(nums) + 1))/2
        sumnums = sum(nums)

        return summation - sumnums

File: 0268. Missing Number/solution.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int real_sum = (nums.size() * (nums.size() + 1)) / 2;

        int nums_sum = 0;

        for (auto& num : nums) nums_sum += num;

        return real_sum - nums_sum;
    }
};